tests/unit_tests/data_steward/tools/retract_data_gcs_test.py

```diff
@@ -1,7 +1,5 @@
 import unittest
 
-from tools import retract_data_gcs as rd
```

Comment:
If you're only testing built-in type conversions, you don't need this.  I think you can remove this entire file.

Reply:
I agree, it's not required.